features/transient-patch.feature

```diff
@@ -0,0 +1,159 @@
+Feature: Patch command available for the transient cache
+
+  Scenario: Nested values from transient can be updated at any depth.
+    Given a WP install
+    And a wp-content/mu-plugins/test-harness.php file:
+      """php
+      <?php
+      $set_foo = function(){
+        set_transient( 'my_key', ['foo' => 'bar'] );
+        set_transient( 'my_key_2', ['foo' => ['bar' => 'baz']] );
+      };
+
+      WP_CLI::add_hook( 'before_invoke:transient patch', $set_foo );
```

Comment:
Can we use `wp transient set` to set these initial transient values? The transient cache is persistent in vanilla WordPress, so we shouldn't need to hook into the command invocation in this manner.

Reply:
Originally I was planing on using `wp transient set` but it is not possible to set array like data directly like what's possible with `wp option add/update`.

I've found another solution by using the `wp eval` command to execute a `set_transient`
```
Given a WP install
    And I run `wp eval "set_transient( 'my_key', ['foo' => 'bar'] );"`
    And I run `wp eval "set_transient( 'my_key_2', ['foo' => ['bar' => 'baz']] );"`
```